stack/keycloak/migration/05_assign-roles.yml

```
@@ -0,0 +1,12 @@
+id: assign roles to users
+author: Default
+realm: local_realm
+changes:
+  - assignRole:
+      user: reader
+      role: reader
+      clientId: local
+  - assignRole:
+      user: writer
+      role: writer
+      clientId: local
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)